src/main/java/uk/gov/hmcts/reform/ccd/document/am/model/DocumentTTLResponse.java

```diff
@@ -0,0 +1,24 @@
+package uk.gov.hmcts.reform.ccd.document.am.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import java.time.LocalDateTime;
+
+@NoArgsConstructor
```

Comment:
try removing @NoArgsConstructor

Reply:
Need no args constructor for one of the tests

Reply:
use this https://github.com/hmcts/idam-java-client/blob/master/src/test/java/uk/gov/hmcts/reform/idam/client/IdamClientTest.java#L88